app/models/patient.rb

```diff
@@ -134,6 +134,15 @@ def most_recent_note
     notes.order('created_at DESC').limit(1).first
   end
 
+  def most_recent_pledge_display_date
```

Comment:
tbh since we don't have pledge implemented and are doing it in a hacky way, we can probably just comment these out or have them return nil for now. Will save us the trouble of writing tests for it !